### PR TITLE
fix(tutorial_texts.js): drop references to "Apply" button in filter step

### PR DESCRIPTION
**Pull request by piperchester**
## Description

Noticed this while perusing #868. Since we don't have the Apply button any longer, this tutorial text is obsolete.

### CODE REVIEW

File: frontend/public/texts/tutorial_texts.js [hunk 1]
```diff
@@ -201,25 +201,23 @@ export default function getTutorialTexts({ hubName, classes, locale }) {
     },
     filter_and_find_tutorial_text: {
       en: `Click on the "Filter" button to filter the projects, for example by location or
-      category. Choose what you want to filter by and click "Apply" to see the results!`,
+      category.`,
       de: `Klicke auf die Schaltfläche "Filter", um die Projekte zu filtern, z. B. nach Ort oder
       Kategorie. Wählen aus, wonach du filtern möchtest und klicke auf "Anwenden", um die Ergebnisse zu sehen!`,
```
Comment (verena-e-p) [inline frontend/public/texts/tutorial_texts.js:206]:
```suggestion
      Kategorie.`,
```
This is the German counterpart of the text you just removed :)

File: frontend/public/texts/tutorial_texts.js [hunk 2]
```diff
@@ -201,25 +201,23 @@ export default function getTutorialTexts({ hubName, classes, locale }) {
...
     },
     filter_and_find_tutorial_text_for_answer_true: {
       en: `Click on the "Filter" button to filter the projects, for example by location or
-      category. Choose what you want to filter by and click "Apply" to see the results!`,
+      category.`,
       de: `Klicke auf die Schaltfläche "Filter", um die Projekte zu filtern, z. B. nach Ort oder
       Kategorie. Wähle aus, wonach du filtern möchtest und klicke auf "Anwenden", um die Ergebnisse zu sehen!`,
```
Comment (verena-e-p) [inline frontend/public/texts/tutorial_texts.js:212]:
```suggestion
      Kategorie.`,
```

File: frontend/public/texts/tutorial_texts.js [hunk 3]
```diff
@@ -201,25 +201,23 @@ export default function getTutorialTexts({ hubName, classes, locale }) {
...
     },
     filter_and_find_tutorial_text_for_answer_soon: {
-      en: `Click on the "Filter" button to filter the projects, for example by location, category or the skills they are looking for. 
-      Choose what you want to filter by and click "Apply" to see the results!`,
+      en: `Click on the "Filter" button to filter the projects, for example by location, category or the skills they are looking for.`,
       de: `Klicke auf die Schaltfläche "Filter", um die Projekte zu filtern, z. B. nach Ort,
       Kategorie oder gesuchten Fähigkeiten. Wähle aus, wonach du filtern möchtest und klicke auf "Anwenden", um die Ergebnisse zu sehen!`,
```
Comment (verena-e-p) [inline frontend/public/texts/tutorial_texts.js:217]:
```suggestion
      Kategorie oder gesuchten Fähigkeiten.`,
```

File: frontend/public/texts/tutorial_texts.js [hunk 4]
```diff
@@ -201,25 +201,23 @@ export default function getTutorialTexts({ hubName, classes, locale }) {
...
     },
     filter_and_find_tutorial_text_for_answer_false: {
-      en: `Click on the "Filter" button to filter the projects, for example by location or category. 
-      Choose what you want to filter by and click "Apply" to see the results!`,
+      en: `Click on the "Filter" button to filter the projects, for example by location or category.`,
       de: `Klicke auf die Schaltfläche "Filter", um die Projekte zu filtern, z. B. nach Ort oder
       Kategorie. Wähle aus, wonach du filtern möchtest und klicke auf "Anwenden", um die Ergebnisse zu sehen!`,
```
Comment (verena-e-p) [inline frontend/public/texts/tutorial_texts.js:222]:
```suggestion
      Kategorie.`,
```

**Review by verena-e-p — APPROVED** [review]

**Comment by piperchester** [comment]
Thanks a ton for the review @verena-e-p! And helping me out with more of my 🇩🇪 :-)

**Pull request merged**